channels/chan_simpleusb.c

```diff
@@ -2404,10 +2407,15 @@ static struct ast_frame *simpleusb_read(struct ast_channel *c)
 			}
 		}
 	}
+
 	/* Scale the input audio.
 	 * o->boost is hardcoded to equal BOOST_SCALE.
 	 * This code is not executed.
+	 * TBR: The below should be removed. Raw audio samples should never be clipped or scaled
```

Comment:
Maybe TODO instead of TBR to use a more standard code acronym?

Reply:
I have updated these comments.